File: Data Structures & Algorithms/concatenation-of-array/submission-0.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        int n = nums.size();
        int j = n;
        vector<int> ans(2*n);
        for(int i=0; i<n;i++){
            ans[i] = nums[i];
            ans[j] = nums[i];
            j++; 
        }
        return ans;
    }
};

File: Data Structures & Algorithms/invert-a-binary-tree/submission-0.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right

class Solution:
    def invertTree(self, root: Optional[TreeNode]) -> Optional[TreeNode]:
        ans = []
        def reverse(curr):
            if curr == None:
                return
            
            temp = curr.left
            curr.left = curr.right
            curr.right = temp
            ans.append(curr.val)
            reverse(curr.left)
            reverse(curr.right)

        
        reverse(root)

        return root